lib/ueberauth/strategy/amco/exceptions.ex

```diff
@@ -5,3 +5,7 @@ end
 defmodule Ueberauth.Strategy.Amco.Exceptions.EmptyAccessTokenSource do
   defexception message: "access_token_source option is required"
 end
+
+defmodule Ueberauth.Strategy.Amco.Exceptions.MissingSiteConfiguration do
+  defexception message: "site configuration option is missed"
```

Comment:
Does this message get appended to something? Asking, because the `s` in site should be capital to start the phrase... if it is the start.

Reply:
Fixed. Thanks.